src/libraries/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.Browser.cs

```diff
@@ -15,6 +15,6 @@ public sealed partial class DriveInfo
         public long TotalFreeSpace => 0;
         public long TotalSize => 0;
 
-        private static string[] GetMountPoints() => Environment.GetLogicalDrives();
+        private static string[] GetMountPoints() => DriveInfoInternal.GetLogicalDrives();
```

Comment:
Is this change required since `Environment.GetLogicalDrives` returns the same value?

Reply:
No not required but removes an extra call because `Environment.GetLogicalDrives()` calls `DriveInfoInternal.GetLogicalDrives();`

Reply:
Why does this extra call matter in this case? 

This is adding a second copy of DriveInfoInternal.Browser.cs that is much worse for Browser in particular than an extra call.

Reply:
@jkotas not sure what you mean, this follows the existing pattern for `DriveInfoInternal.Unix.cs` that is already there.

Reply:
When having a choice, I think we should be optimizing Browser for smaller footprint, not for tiny bit better throughput of APIs that are slow by design and are very unlikely to be used in Browser.

I believe that the extra copy of `DriveInfoInternal.Browser.cs` is causing more harm here for browser than the extra call.

This does not matter much since the difference is small and all this code is going to be removed by the linker for browser anyway.

Reply:
Oh I see, you meant the extra copy of the implementation that is compiled into the library, I thought you meant some duplicate file :)